README.md

```diff
@@ -1,8 +1,8 @@
 # Google Compute Engine Persistent Disk CSI Driver
 
-WARNING: This driver is beta and should not be used in performance critical applications
+WARNING: Manual deployment of this driver to your cluster is not recommended. Instead users should user GKE to automatically deploy and manage the GCE PD CSI Driver (see [GKE Docs](https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/gce-pd-csi-driver)).
 
-DISCLAIMER: This is not an officially supported Google product
+DISCLAIMER: Manual deployment of the driver to your cluster is not an officially supported by Google.
```

Comment:
clarify on GKE? I guess it's also not officially supported for non-GKE users too?

remove "an"

Reply:
Ya, it's not supported in either case, so I'll leave GKE out of this line. Added it to the one above.